### ISSUE TITLE
为什么finetune/classifier.py中设置了is_predict时返回的是pyreader,probs和feed_targets_name而不是pyreader和global_vars？

### ISSUE THREAD

**Issue by LukeALee** [PaddlePaddle/ERNIE#198]
ERNIE/finetune/classifier.py中定义了    

       if is_prediction:
        probs = fluid.layers.softmax(logits,name="probs")
        feed_targets_name = [
            src_ids.name, pos_ids.name, sent_ids.name, input_mask.name
        ]
        return pyreader, probs, feed_targets_name

而train、test、eval时返回的都是pyreader和global_vars，此时可以通过global_vars["probs"].name获取想要fetch的变量，那么如果返回的是pyreader, probs, feed_targets_name，在写predict程序时如何获取想要fetch的probs和labels呢？

**Comment by tianxin1860** [comment]
> ERNIE/finetune/classifier.py中定义了
> 
> ```
>    if is_prediction:
>     probs = fluid.layers.softmax(logits,name="probs")
>     feed_targets_name = [
>         src_ids.name, pos_ids.name, sent_ids.name, input_mask.name
>     ]
>     return pyreader, probs, feed_targets_name
> ```
> 
> 而train、test、eval时返回的都是pyreader和global_vars，此时可以通过global_vars["probs"].name获取想要fetch的变量，那么如果返回的是pyreader, probs, feed_targets_name，在写predict程序时如何获取想要fetch的probs和labels呢？

在 predict 阶段 fetch probs 的方式和在训练阶段 fetch probs 的方法是一样的，分类任务的 predict 脚本已经 release: [https://github.com/PaddlePaddle/LARK/pull/199](https://github.com/PaddlePaddle/LARK/pull/199)
